FIPS/src/test/java/net/snowflake/client/jdbc/ConnectionFipsIT.java

```diff
@@ -289,7 +290,12 @@ public void testConnectUsingKeyPair() throws Exception {
     DriverManager.getConnection(uri, properties).close();
   }
 
+  /**
+   * Test case for connecting with FIPS and executing a query.
+   * Currently ignored execution on GCP due to exception thrown "SSlException Could not generate XDH keypair"
```

Comment:
should we add an issue to fix this problem?

Reply:
An issue was reported. https://snowflakecomputing.atlassian.net/browse/SNOW-1496324